src/dropdown.ts

```diff
@@ -1544,8 +1581,6 @@ export default class Dropdown extends LitElement implements FormControl {
           // when using VoiceOver.
           event.preventDefault();
 
-          this.isFiltering = false;
```

Comment:
Now set once in activeOption.selected = this.multiple
            ? !this.activeOption.selected
            : t.